### PR TITLE
Hi-C assay template

### PR DESCRIPTION
**Pull request by cw-dvr8**

### CODE REVIEW

File: PsychENCODE/assay_HiC_schema.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.2-dev/current_annotations_schemas/ngs.json#/definitions/readLength"
+    },
+    "runType": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.2-dev/current_annotations_schemas/ngs.json#/definitions/runType"
+    },
+    "totalReads": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.2-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/totalReads"
+    },
+    "validHiCContacts": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.2-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/validHiCContacts"
```
Comment (kelshmo) [inline PsychENCODE/assay_HiC_schema.json]: This ref currently does not display this key - is that because it will update after this PR is merged?

**Comment by cw-dvr8** [comment]
@kelshmo It will once the branch is merged.  The validHiCContact was added into one of the other files that's in this PR.

**Comment by kelshmo** [comment]
> @kelshmo It will once the branch is merged. The validHiCContact was added into one of the other files that's in this PR.

@cw-dvr8 How do you track version release? Do you follow the syntax of adding keys or values would be a minor release? In that case, the version should increment to 0.0.3. You could adopt the same system of synapseAnnotations - [documented here](https://github.com/sage-bionetworks/synapseannotations#release-versioning-annotations).

**Comment by cw-dvr8** [comment]
It's still up in the air how we're doing releases, because using the release number means that I have to update the remote reference in every schema every time we change it and I need to automate that process or else it becomes too time consuming.  For now I merge to the master branch, and then merge to the current version in the gh-pages branch for publishing.

**Pull request closed without merging**